content/pages/health-care/health-conditions.md

```diff
@@ -59,19 +63,40 @@ At VA, we offer many services and programs for Veterans with specific needs—li
 
 **[Get an overview of what’s covered by VA health care](/health-care/about-va-health-care/).**
 
+</div>
+</div>
+</div>
+
 ------
 
-#### What if I need help understanding my benefits or accessing services?
+<div itemscope itemtype="http://schema.org/Question">
+
+<h4 itemprop="name">What if I need help understanding my benefits or accessing services?</h4>
+<div itemprop="acceptedAnswer" itemscope itemtype="http://schema.org/Answer">
+<div itemprop="text">
+
 
 If you want to know more about your VA health care benefits, your health care team, and where you’ll go for care, we can help. [Learn more about your VA health care coverage](/health-care/about-va-health-care/).
 
 If you need help accessing services, call our toll-free hotline at 1-877-222-VETS (<a href="tel:+1-877-222-8387">1-877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (<abbr title="eastern time">ET</abbr>).
 
+</div>
+</div>
+</div>
 <br>
 
-#### Can I manage my VA health and benefits online?
+<div itemscope itemtype="http://schema.org/Question">
+
+<h4 itemprop="name">Can I manage my VA health and benefits online?</h4>
+<div itemprop="acceptedAnswer" itemscope itemtype="http://schema.org/Answer">
+<div itemprop="text">
 
 Yes. If you’re signed up for the VA health care program, you can manage your VA health and benefits online through Vets.gov: <br>
```

Comment:
When this page renders, there's a double-grey rule at the bottom -- I'm not sure this is meant to be there; would you mind removing at least one of the two grey lines please? 

<img width="758" alt="screen shot 2017-11-16 at 10 01 43 am" src="https://user-images.githubusercontent.com/4895604/32898095-38df43f4-cab5-11e7-93fb-9de3c687683a.png">